src/main/kotlin/com/sourcegraph/cody/history/state/HistoryState.kt

```diff
@@ -5,10 +5,8 @@ import com.intellij.util.xmlb.annotations.OptionTag
 
 class HistoryState : BaseState() {
 
-  @get:OptionTag(tag = "chats", nameAttribute = "") var chats: MutableList<ChatState> by list()
+  @get:OptionTag(tag = "accountHistories", nameAttribute = "")
+  var accountHistories: MutableList<AccountHistoryState> by list()
 
   @get:OptionTag(tag = "defaultLlm", nameAttribute = "") var defaultLlm: LLMState? by property()
```

Comment:
Default LLM should also be per account, so we need to move it to `AccountHistoryState`